plugin/core/types.py

```diff
@@ -71,6 +72,37 @@ def matches_pattern(path: str, patterns: Any) -> bool:
     return False
 
 
+def sublime_pattern_to_glob(pattern: str, is_directory_pattern: bool, root_path: Optional[str] = None) -> str:
```

Comment:
Do you see a benefit to split this function into two separate functions? (to remove the is_directory_pattern flag)
- sublime_pattern_to_folder_glob
- sublime_pattern_to_file_glob

Reply:
That would result in duplication of logic.